### PR TITLE
Added handling of network connectivity status

### PR DESCRIPTION
**Pull request by vincentcastagna**
- Added Xamarin Essentials v1.1.0 to project
- Updated Uno to 1.45.0-dev.1813 
- Added a new property to TaskNotifier to identify network issues 
- Created a converter to display the right issue and avoid code duplication in xaml views.

### CODE REVIEW

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.nventive.uno.azuredevops" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="28" />
-	<application android:label="uado" android:icon="@mipmap/icon" android:allowBackup="false"></application>
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml]: Why remove this?

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Framework/Tasks/TaskNotifier.cs [hunk 2]
```diff
@@ -60,6 +63,12 @@ private void RunTask(Task taskToExecute)
 			taskToExecute.ContinueWith(
 				task =>
 				{
+#if !__WASM__
+					if (Connectivity.NetworkAccess != NetworkAccess.Internet)
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Framework/Tasks/TaskNotifier.cs]: This takes into account both Wifi and Cellular network, right?

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml (hunk 1 above)
Comment (vincentcastagna) [reply src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml]: Not intended, seems manifest permission in VS removed it :O

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Framework/Tasks/TaskNotifier.cs (hunk 2 above)
Comment (vincentcastagna) [reply src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Framework/Tasks/TaskNotifier.cs]: Yes it is !

File: src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml [hunk 3]
```diff
@@ -2,4 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.nventive.uno.azuredevops" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="28" />
 	<application android:label="uado" android:icon="@mipmap/icon" android:allowBackup="false"></application>
-</manifest>
\ No newline at end of file
+	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<application android:label="uado" android:allowBackup="false"></application>
```
Comment (andreaSflores18) [inline src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml]: Duplicate of line 4

**Pull request merged**